dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/FATSuite.java

```diff
@@ -69,7 +69,8 @@ public class FATSuite extends TestContainerSuite {
             } else {
                 repeat = RepeatTests.with(new EmptyAction().fullFATOnly())
                                 .andWith(FeatureReplacementAction.EE9_FEATURES().fullFATOnly())
-                                .andWith(FeatureReplacementAction.EE10_FEATURES());
+                                .andWith(FeatureReplacementAction.EE10_FEATURES().fullFATOnly())
+                                .andWith(FeatureReplacementAction.EE11_FEATURES());
```

Comment:
It would be good to use the conditionalFullFAT unless there is a reason we're not here. See https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.el.3.0_fat/fat/src/com/ibm/ws/el/fat/FATSuite.java for an example